### ISSUE TITLE
API is not instantiated in SimpleAlbum after getTrack call

### ISSUE THREAD

**Issue by adamint** [adamint/spotify-web-api-kotlin#237]
UninitializedPropertyAccessException {message_8yp7un$_0: "lateinit property api has not been initialized

**Comment by andb3** [comment]
This also is happening in SimplePlaylist for me, in case that hadn't been logged yet

**Comment by adamint** [comment]
Thank you for the info! For now, please use the corresponding api endpoint directly. This is on hold until after #238 is merged, because it's introducing a new deserialization strategy.
